make/common/rsyslog/Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM library/photon:1.0
+FROM vmware/photon:1.0-20170928
```

Comment:
The bug of vmware/photon has been fixed, so you can use 1.0 now.

Reply:
Nice.